.vscode/settings.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
This looks like a local VS Code file. should it really have been checked in?

Reply:
no. it shouldn't. Will remove the file and add an entry to `.gitignore`.

Haven't found the time to fix the issue with the unit test yet :-/ Trying to analyze & fix it next weekend.